charon-ml/src/LlbcOfJson.ml

```diff
@@ -181,43 +181,42 @@ let split_global (gid_conv : global_id_converter) global :
   (global_decl, fun_decl)
 
 let crate_of_json (js : json) : (crate, string) result =
-  combine_error_msgs js __FUNCTION__
```

Comment:
The `combine_error_msgs` is not used anymore?

Reply:
This is only for the version mismatch error: we don't want to dump the parsed AST in that case.